app/lib/concerns/versioning_helpers.rb

```diff
@@ -14,4 +14,14 @@ def minor_version_number(version)
   def version_number(version)
     "#{major_version_number(version)}.#{minor_version_number(version)}"
   end
+
+  def api_docs_version_navigation_items
```

Comment:
does this logic belong here? I think because its mainly a FE concern we can move it and the method below elsewhere, maybe the presenter?